src/main/java/com/pdffiller/client/utils/MultipartHelper.java

```diff
@@ -0,0 +1,88 @@
+package com.pdffiller.client.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.file.Files;
+import java.security.SecureRandom;
+import java.util.HashMap;
+
+public class MultipartHelper {
+  private static final String CRLF = "\r\n";
+  private File file;
+  private String boundary;
+  private byte[] message;
+
+  public File getFile() {
+    return file;
+  }
+
+  public void setFile(File file) {
+    this.file = file;
+  }
+
+  public byte[] getMessage() {
+    return message;
+  }
+
+  public MultipartHelper(String file) {
+    this.file = new File(file);
+    boundary = generateBoundary();
+    message = formRequest();
+  }
+
+  private String generateBoundary() {
+    SecureRandom random = new SecureRandom();
+    return new BigInteger(20, random).toString();
+  }
+
+  private String getMime() {
+    String mime = null;
+    try {
+      mime = Files.probeContentType(file.toPath());
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+
+    return mime;
+  }
+
+  public byte[] getFileContent() {
+    byte[] content = new byte[0];
+    try {
+      content = Files.readAllBytes(file.toPath());
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+    return content;
+  }
+
+  public byte[] formRequest() {
+    String message1 = "--" + boundary + CRLF;
+
+    message1 += "Content-Disposition: form-data; name=\"file\";" +
```

Comment:
What is a "message1" , "content" and "message2"?

Let's call it messageHeader, messageContent and messageBoundary. 
Remember, this code will be read publicly.

I recommend using StringBuilder instead of concatenation + 

StringBuilder sb = new StringBuilder();
str = sb.append(str1).append(str2).append(str3)

+ creates new object every time it is used. Modern Java compilers convert + to StringBuilder on the background